coala_quickstart/green_mode/find_globs.py

```diff
@@ -0,0 +1,185 @@
+import os
+from fnmatch import fnmatch
+
+
+def find_globs_from_files(node, initials, project_dir, files, ignore_list,
```

Comment:
This should go into utilities

Reply:
He is putting it here because this entire module is being added to the no-cover list

Reply:
If the code is still long, add small 1-2 line comments above different sections of code explaining what they do